automation/jleechanorg_pr_automation/tests/test_minimax_preflight_env.py

```
@@ -0,0 +1,114 @@
+        self, mock_which, mock_validate
+    ):
+        """Verify MINIMAX_API_KEY is mapped to ANTHROPIC_API_KEY in validation env."""
+        # Setup mocks
+        mock_which.return_value = "/usr/bin/claude"
+        mock_validate.return_value = MagicMock(success=True, output_file=None)
+
+        # Set MINIMAX_API_KEY in environment
+        test_api_key = "test-minimax-key-12345"
+        with patch.dict(os.environ, {"MINIMAX_API_KEY": test_api_key}, clear=False):
+            # Call the validation method
+            result = self.monitor._run_two_phase_cli_validation(
```

Ruff (0.15.0)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)